Bundle/DependencyInjection/EnqueueBridgeExtension.php

```diff
@@ -31,24 +31,24 @@ public function load(array $configs, ContainerBuilder $container)
             return;
         }
 
+        $contextManager = new Definition(AmqpContextManager::class, array(
+            new Reference('enqueue.transport.default.context'),
+        ));
+
         $receiverDefinition = new Definition(QueueInteropReceiver::class, array(
             new Reference('messenger.transport.default_decoder'),
-            new Definition(AmqpContextManager::class, array(
-                new Reference('enqueue.transport.default.context'),
-            )),
+            $contextManager,
+            $config['queue'],
```

Comment:
Why removing the default value?

Reply:
because it's mandatory in the configuration.